File: 742-to-lower-case/to-lower-case.py
class Solution:
    def toLowerCase(self, s: str) -> str:
        res=''
        for ch in s:
            if ord(ch)>=65 and ord(ch)<=90:
                res=res+chr(ord(ch)+32)
            else:
                res=res+ch
        return res
        '''return s.lower()'''

        

File: 742-to-lower-case/to-lower-case.java
class Solution {
    public String toLowerCase(String s) {
       String res="";

       for(int i=0;i<s.length();i++){
           char ch=s.charAt(i);

           if(ch>=65 && ch<=90){
                res=res+(char)(ch+32);
           }
           else{
                res=res+ch;
           }
       }
       return res;
    }
}